xbmc/settings/AdvancedSettings.cpp

```diff
@@ -1294,6 +1299,11 @@ void CAdvancedSettings::GetCustomTVRegexps(TiXmlElement *pRootElement, SETTINGS_
           {
             bByDate = true;
           }
+          std::string byTitle = XMLUtils::GetAttribute(pRegExp->ToElement(), "bytitle");
+          if (byTitle == "true")
+          {
+            bByTitle = true;
```

Comment:
can be simplified similar to `bByTitle = (byTitle == "true");`

Reply:
or also:
`bByTitle = XMLUtils::GetAttribute(pRegExp->ToElement(), "bytitle") == "true";`
if the std::string it is not needed